### PR TITLE
Update French Translation for Media modules

### PR DESCRIPTION
**Pull request by Vilsafur**

### CODE REVIEW

**Comment by joshbrw** [comment]
@nWidart Can you proof read please? They look good to me though.

File: Resources/lang/media/fr/media.php [hunk 1]
```diff
@@ -13,6 +13,11 @@
         'width' => 'Largeur',
         'height' => 'Hauteur',
     ],
+    'form' => [
+        'alt_attribute' => 'Attribut alt',
```
Comment (nWidart) [inline Resources/lang/media/fr/media.php:17]: Maybe Attribut alternatif ?

Comment (Vilsafur) [inline Resources/lang/media/fr/media.php:17]: if it's for users comprehension, it's "Attribut de remplacement".
Otherwise, it's "Attribut alt"

Comment (nWidart) [inline Resources/lang/media/fr/media.php:17]: Fair enough.

**Pull request merged**